components/tidb_query/src/codec/datum.rs

```diff
@@ -290,7 +290,7 @@ impl Datum {
             Datum::I64(i) => format!("{}", i),
             Datum::U64(u) => format!("{}", u),
             Datum::F64(f) => format!("{}", f),
-            Datum::Bytes(ref bs) => String::from_utf8(bs.to_vec())?,
+            Datum::Bytes(ref bs) => unsafe { String::from_utf8_unchecked(bs.to_vec()) },
```

Comment:
Could you please explain why we should not return a `Result` instead? From my perspective, if we just print the bytes we use use `debug` format instead of converting it to a `UTF-8` string without any check.

Reply:
I think if we use from_utf8_unchecked we can just return the String value.

Reply:
I think it's OK to return a string directly.

Reply:
> I think it's OK to return a string directly.

Should we use the from_utf8_unchecked?  I don't know this change is reasonable.

Reply:
@breeswish PTAL

Reply:
Sorry, I have discussed with him. I will refactor it tonight.

Reply:
refactored.